### PR TITLE
Update cla.yml

### PR DESCRIPTION
**Pull request by lipmaneth**
previous cla gh action didn't appear to be working correctly. Updating the CLA to retry.

## Description

The previous CLA was causing failing checks and not displaying the correct info to the user. Trying again by updating the CLA.yml file.

## Additional Information

- [x] I read the [contributing docs](https://github.com/GuardianUI/guardianui/blob/main/CONTRIBUTING.md) (if this is your first contribution)

### CODE REVIEW

**Comment by lipmaneth** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by lipmaneth** [comment]
@0xLienid could you take a look at this? Had Jem on it but doesn't look like he was able to yet.

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -5,33 +5,39 @@ on:
...
           # the below token should have repo scope and must be manually added by you in the repository's secret
-          PERSONAL_ACCESS_TOKEN : ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+          # This token is required only if you have configured to store the signatures in a remote repository/organization
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
         with:
           path-to-signatures: 'signatures/version1/cla.json'
-          path-to-document: 'https://github.com/GuardianUI/guardianui/blob/main/CLA.md' # e.g. a CLA or a DCO document
+          path-to-document: 'https://github.com/cla-assistant/github-action/blob/master/SAPCLA.md' # e.g. a CLA or a DCO document
           # branch should not be protected
           branch: 'main'
```
Comment (0xLienid) [inline .github/workflows/cla.yml:31]: main is a protected branch which may be causing issues with this

**Comment by 0xLienid** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by lipmaneth** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**